env-files/tensorflow/generic_tf.sh

```diff
@@ -88,9 +88,9 @@ pip3 install --no-cache-dir  tf_keras==2.16.*
 
 # Install Pov4ML
 if [[ "$OSTYPE" =~ ^darwin ]] ; then
-  pip install "prov4ml[apple]@git+https://github.com/matbun/ProvML@main" || exit 1
+  pip install "prov4ml[apple,nvidia]@git+https://github.com/matbun/ProvML@new-main" || exit 1
```

Comment:
This might be fine, but I do wonder how it is possible for `apple` and `nvidia` to be compatible extras here.

Reply:
Good point. Can you install `nvitop` on apple devices? If not, I will remove it